addons/binding/org.openhab.binding.lightify/src/main/java/org/openhab/binding/lightify/handler/DeviceHandler.java

```diff
@@ -0,0 +1,208 @@
+import static org.openhab.binding.lightify.internal.LightifyConstants.TW_CHANNEL_ID_DIMMER;
+import static org.openhab.binding.lightify.internal.LightifyConstants.TW_CHANNEL_ID_POWER;
+import static org.openhab.binding.lightify.internal.LightifyConstants.TW_CHANNEL_ID_TEMPERATURE;
+
+/**
+ * <p>The {@link org.eclipse.smarthome.core.thing.binding.ThingHandler} implementation to handle commands
+ * for a OSRAM Lightify light bulb / stripe or zone (group of lights).</p>
+ * <p>Commands are sent through the Lightify gateway the device is paired to. The connection link, managed
+ * by the assigned {@link GatewayHandler} will be used to forward the command to.</p>
+ *
+ * @author Christoph Engelbert (@noctarius2k) - Initial contribution
+ */
+public class DeviceHandler extends BaseThingHandler {
+
+    /**
+     * Supported {@link ThingTypeUID}s for this handler
+     */
+    public static final Set<ThingTypeUID> SUPPORTED_TYPES = Collections.unmodifiableSet(new HashSet<ThingTypeUID>() {
+        {
+            add(THING_TYPE_LIGHTIFY_BULB_SB);
+            add(THING_TYPE_LIGHTIFY_BULB_TW);
+            add(THING_TYPE_LIGHTIFY_BULB_RGBW);
+            add(THING_TYPE_LIGHTIFY_ZONE);
+        }
+    });
+
+    private final Logger logger = LoggerFactory.getLogger(DeviceHandler.class);
+
+    private volatile ScheduledFuture<?> updateTaskRegistration;
+
+    public DeviceHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        Runnable statusUpdater = () -> {
+            GatewayHandler gatewayHandler = getGatewayHandler();
+            LightifyLink lightifyLink = gatewayHandler.getLightifyLink();
+            LightifyLuminary luminary = getLuminary();
+            if (luminary instanceof LightifyLight) {
+                lightifyLink.performStatusUpdate(luminary, this::updateState);
+            }
+        };
+
+        updateTaskRegistration = scheduler.scheduleWithFixedDelay(statusUpdater, 1, 10, TimeUnit.SECONDS);
+    }
+
+    @Override
+    public void dispose() {
+        updateTaskRegistration.cancel(true);
+        super.dispose();
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Command: {}", command);
+        switch (channelUID.getId()) {
+            case RGBW_CHANNEL_ID_POWER:
+            case SB_CHANNEL_ID_POWER:
+            case TW_CHANNEL_ID_POWER:
+                handlePowerSwitch(command);
+                break;
+            case RGBW_CHANNEL_ID_DIMMER:
+            case SB_CHANNEL_ID_DIMMER:
+            case TW_CHANNEL_ID_DIMMER:
+                handleDimmer(command);
+                break;
+            case RGBW_CHANNEL_ID_TEMPERATURE:
+            case TW_CHANNEL_ID_TEMPERATURE:
+                handleTemperature(command);
+                break;
+            case RGBW_CHANNEL_ID_COLOR:
+                handleColor(command);
+        }
```

Comment:
See https://www.eclipse.org/smarthome/documentation/development/bindings/how-to.html#handling-commands-and-updating-the-state